### PR TITLE
Reproducability

### PR DESCRIPTION
**Pull request by stroblme**
Fixes the reproducability issue by introducing a seed as parameter in 'training.yml' that goes into the 'torch.manual_seed' method ([see docs](https://pytorch.org/docs/stable/notes/randomness.html)) within the instructor init.

### CODE REVIEW

**Comment by stroblme** [comment]
Quick result from two runs with identical parameters resulting in identical metric values. Intermediate values match as well; plot omitted.
image

**Review by cDenius — APPROVED** [review]
yup, looks good :+1:
